### ISSUE TITLE
Add inventory sub-command to return inventory JSON content

### ISSUE THREAD

**Issue by jlaska** [ansible/tower-cli#89]
Add @wenottingham for additional comments

**Comment by AlanCoding** [comment]
My first question is how this request is different from running the command:

> tower-cli inventory list --format=json

This returns the JSON output which contains all of the inventory records. Once we get more people to weigh in on this issue, I predict that the requirements will go beyond this. For now, I just want to get this out there to establish a baseline functionality which already exists and is almost trivial to use.

**Comment by jlaska** [comment]
@AlanCoding the provided command would be expected to list the `json` output of the `/api/v1/inventories/N/` endpoint from Tower.

You want the equivalent of `/api/v1/inventories/N/script/` surfaced through `tower-cli inventory script`?

**Comment by wenottingham** [comment]
RIght, we'd want the json form of a particular inventory, including variables - it would be more 'tower-cli inventory get 3 --format=json'

**Comment by AlanCoding** [comment]
Thanks, I'm understanding better now. `/api/v1/inventories/N/` is a completely different animal from `/api/v1/inventories/N/script/`. I picture this being returned by a command like `tower-cli inventory_script get N --format=json`. Maybe that syntax could use some tweaking as well. Depending on what the usage scenario is, we might want something more than a get request, but I'm not sure yet.

**Comment by cchurch** [comment]
Make sure you use `/api/v1/inventories/N/script/?hostvars=1` to return the host variables in the response.
